zipkin-lens/package.json

```diff
@@ -52,12 +52,10 @@
     "jest-standard-reporter": "^1.0.1",
     "mini-css-extract-plugin": "^0.7.0",
     "node-fetch": "^2.3.0",
-    "node-sass": "^4.11.0",
```

Comment:
node-sass is a quite common problem area when people are building.. are we no longer using this? or is it just no longer an explicit dep?

Reply:
We're no longer using it with this PR - I think there was a ton of SCSS a while ago but now none :)